### DISCUSSION TITLE
Add Jolt Transformation Feature for jsoncrack

### DISCUSSION THREAD

**Discussion by emincanoguz11** [AykutSarac/jsoncrack.com#186] [Ideas]
Hi everyone,

This program is awesome. Thanks to the contributors. But I have a idea for this program. Sometimes we must change the json structure. (For example move root element to main element etc). Jolt Transformation is very quick solution for this reasons and maybe we will see this result to the image.

What about this feature? 

Best wishes,
Emin Can

**Comment by emincanoguz11** [comment]
@AykutSarac what do you think about this feature?

**Comment by AykutSarac** [comment]
Hello,

As JSON Crack's one of the primary goal is to keep the app simple and single purpose as much as possible, Jolt transformation seem to be currently off the plan, for now. Thanks for the suggestion, please keep them coming! :)
